package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
     "name": "file-downloader",
-    "version": "1.0.11",
+    "version": "1.0.12",
```

Comment:
I wonder how come package-lock json changes show up without package.json changes ? Am I missing something? Please let me know. Thanks.

Reply:
Never mind got it, someone didn't commit the correct packag-lock.json last time.